Fix:
dummy.c:617:28: error: implicit declaration of function 'time' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
netmsg.c:63:45: error: implicit declaration of function 'time' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
nntp.c:1261:39: error: implicit declaration of function 'time' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
unix.c:231:2: error: implicit declaration of function 'utime' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
mbx.c:303:7: error: implicit declaration of function 'utime' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
mmdf.c:382:2: error: implicit declaration of function 'utime' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
tenex.c:206:2: error: implicit declaration of function 'utime' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
tenex.c:656:13: error: implicit declaration of function 'time' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
mtx.c:199:2: error: implicit declaration of function 'utime' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
mh.c:1282:3: error: implicit declaration of function 'utime' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
mx.c:1286:3: error: implicit declaration of function 'utime' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
mlock.c:60:3: error: implicit declaration of function 'write' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
mlock.c:75:31: error: implicit declaration of function 'getegid' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
mlock.c:94:3: error: implicit declaration of function 'chdir' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
mlock.c:113:30: error: implicit declaration of function 'time' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
mlock.c:114:7: error: implicit declaration of function 'unlink' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
mlock.c:123:23: error: implicit declaration of function 'getpid' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
mlock.c:125:5: error: implicit declaration of function 'gethostname' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
mlock.c:132:7: error: implicit declaration of function 'close' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
mlock.c:138:11: error: implicit declaration of function 'link' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
mlock.c:161:20: error: implicit declaration of function 'sleep' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
mlock.c:169:3: error: implicit declaration of function 'read' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
dmail.c:162:35: error: implicitly declaring library function 'isdigit' with type 'int (int)' [-Werror,-Wimplicit-function-declaration]
tmail.c:171:37: error: implicitly declaring library function 'isdigit' with type 'int (int)' [-Werror,-Wimplicit-function-declaration]
--- src/c-client/netmsg.c.orig	2011-07-22 19:20:18.000000000 -0500
+++ src/c-client/netmsg.c	2020-11-21 17:41:51.000000000 -0600
@@ -28,6 +28,7 @@
 
 
 #include <stdio.h>
+#include <time.h>
 #include <errno.h>
 extern int errno;		/* just in case */
 #include "c-client.h"
--- src/c-client/nntp.c.orig	2011-07-22 19:20:18.000000000 -0500
+++ src/c-client/nntp.c	2020-11-21 17:42:51.000000000 -0600
@@ -29,6 +29,7 @@
 
 #include <ctype.h>
 #include <stdio.h>
+#include <time.h>
 #include "c-client.h"
 #include "newsrc.h"
 #include "netmsg.h"
--- src/dmail/dmail.c.orig	2011-07-22 19:19:57.000000000 -0500
+++ src/dmail/dmail.c	2020-11-21 22:18:49.000000000 -0600
@@ -26,6 +26,7 @@
  * Last Edited:	30 October 2008
  */
 
+#include <ctype.h>
 #include <stdio.h>
 #include <pwd.h>
 #include <errno.h>
--- src/mlock/mlock.c.orig	2011-07-22 19:19:57.000000000 -0500
+++ src/mlock/mlock.c	2020-11-21 22:17:32.000000000 -0600
@@ -40,6 +40,8 @@
 #include <netdb.h>
 #include <ctype.h>
 #include <string.h>
+#include <time.h>
+#include <unistd.h>
 
 #define LOCKTIMEOUT 5		/* lock timeout in minutes */
 #define LOCKPROTECTION 0664
--- src/osdep/unix/dummy.c.orig	2011-07-22 19:20:10.000000000 -0500
+++ src/osdep/unix/dummy.c	2020-11-21 17:41:12.000000000 -0600
@@ -35,6 +35,7 @@
 #include "osdep.h"
 #include <pwd.h>
 #include <sys/stat.h>
+#include <time.h>
 #include "dummy.h"
 #include "misc.h"
 
--- src/osdep/unix/mbx.c.orig	2011-07-22 19:20:11.000000000 -0500
+++ src/osdep/unix/mbx.c	2020-11-21 17:45:31.000000000 -0600
@@ -43,6 +43,7 @@
 #include <pwd.h>
 #include <sys/stat.h>
 #include <sys/time.h>
+#include <utime.h>
 #include "misc.h"
 #include "dummy.h"
 #include "fdstring.h"
--- src/osdep/unix/mh.c.orig	2011-07-22 19:20:09.000000000 -0500
+++ src/osdep/unix/mh.c	2020-11-21 22:14:00.000000000 -0600
@@ -36,6 +36,7 @@
 #include <pwd.h>
 #include <sys/stat.h>
 #include <sys/time.h>
+#include <utime.h>
 #include "misc.h"
 #include "dummy.h"
 #include "fdstring.h"
--- src/osdep/unix/mmdf.c.orig	2011-07-22 19:20:10.000000000 -0500
+++ src/osdep/unix/mmdf.c	2020-11-21 17:46:16.000000000 -0600
@@ -34,6 +34,7 @@
 #include "osdep.h"
 #include <time.h>
 #include <sys/stat.h>
+#include <utime.h>
 #include "pseudo.h"
 #include "fdstring.h"
 #include "misc.h"
--- src/osdep/unix/mtx.c.orig	2011-07-22 19:20:10.000000000 -0500
+++ src/osdep/unix/mtx.c	2020-11-21 21:02:16.000000000 -0600
@@ -43,6 +43,7 @@
 #include <pwd.h>
 #include <sys/stat.h>
 #include <sys/time.h>
+#include <utime.h>
 #include "misc.h"
 #include "dummy.h"
 #include "fdstring.h"
--- src/osdep/unix/mx.c.orig	2011-07-22 19:20:09.000000000 -0500
+++ src/osdep/unix/mx.c	2020-11-21 22:15:05.000000000 -0600
@@ -36,6 +36,7 @@
 #include <pwd.h>
 #include <sys/stat.h>
 #include <sys/time.h>
+#include <utime.h>
 #include "misc.h"
 #include "dummy.h"
 #include "fdstring.h"
--- src/osdep/unix/tenex.c.orig	2011-07-22 19:20:10.000000000 -0500
+++ src/osdep/unix/tenex.c	2020-11-21 17:47:14.000000000 -0600
@@ -46,6 +46,8 @@
 #include "mail.h"
 #include "osdep.h"
 #include <sys/stat.h>
+#include <time.h>
+#include <utime.h>
 #include "misc.h"
 #include "dummy.h"
 
--- src/osdep/unix/unix.c.orig	2011-07-22 19:20:10.000000000 -0500
+++ src/osdep/unix/unix.c	2020-11-21 17:44:41.000000000 -0600
@@ -46,6 +46,7 @@
 #include "osdep.h"
 #include <time.h>
 #include <sys/stat.h>
+#include <utime.h>
 #include "unix.h"
 #include "pseudo.h"
 #include "fdstring.h"
--- src/tmail/tmail.c.orig	2011-07-22 19:19:58.000000000 -0500
+++ src/tmail/tmail.c	2020-11-21 22:18:44.000000000 -0600
@@ -26,6 +26,7 @@
  * Last Edited:	30 October 2008
  */
 
+#include <ctype.h>
 #include <stdio.h>
 #include <pwd.h>
 #include <errno.h>
